modules/drivers/databricks-jdbc/test/metabase/test/data/databricks_jdbc.clj

```diff
@@ -0,0 +1,163 @@
+                                   :type/Time                   "TIMESTAMP" #_"TIMESTAMP_NTZ"
+                                   :type/DateTime               "TIMESTAMP"  #_"TIMESTAMP_NTZ"
+                                   :type/DateTimeWithLocalTZ    "TIMESTAMP"
+                                   :type/DateTimeWithTZ         "TIMESTAMP"
+                                   :type/DateTimeWithZoneOffset "TIMESTAMP"
+                                   :type/Decimal                "DECIMAL"
+                                   :type/Float                  "DOUBLE"
+                                   :type/Integer                "INTEGER"
+                                   :type/Text                   "STRING"}]
+  (defmethod sql.tx/field-base-type->sql-type [:databricks-jdbc base-type] [_driver _base-type] database-type))
+
+(doseq [feature [:test/time-type
+                 :test/timestamptz-type
+                 :test/dynamic-dataset-loading]]
+  (defmethod driver/database-supports? [:databricks-jdbc feature]
+    [_driver _feature _database]
+    false))
+
+(defmethod tx/dbdef->connection-details :databricks-jdbc
+  [_driver _connection-type {:keys [database-name] :as _dbdef}]
+  (merge
+   {:host      (tx/db-test-env-var-or-throw :databricks-jdbc :host)
+    :token     (tx/db-test-env-var-or-throw :databricks-jdbc :token)
+    :http-path (tx/db-test-env-var-or-throw :databricks-jdbc :http-path)
+    :catalog   (tx/db-test-env-var-or-throw :databricks-jdbc :catalog)}
+   ;; Databricks' namespace model: catalog, schema, table. With current implementation user can add all schemas
+   ;; in catalog on one Metabase database connection. Following expression generates schema filters so only one schema
+   ;; is treated as a Metabase database, for compatibility with existing tests.
+   (when (string? (not-empty database-name))
+     {:schema-filters-type "inclusion"
+      :schema-filters-patterns database-name})))
+
+(defn- existing-databases
+  "Set of databases that already exist. Used to avoid creating those"
+  []
+  (sql-jdbc.execute/do-with-connection-with-options
+   :databricks-jdbc
+   (->> (tx/dbdef->connection-details :databricks-jdbc nil nil)
+        (sql-jdbc.conn/connection-details->spec :databricks-jdbc))
+   nil
+   (fn [^java.sql.Connection conn]
+     (into #{} (map :databasename) (jdbc/query {:connection conn} ["SHOW DATABASES;"])))))
+
+(defmethod tx/dataset-already-loaded? :databricks-jdbc
+  [_driver dbdef]
+  (contains? (existing-databases) (:database-name dbdef)))
+
+;; TODO: Disable when done with development!
+(def ^:private ^:dynamic *allow-database-creation*
+  "Same approach is used in Databricks driver as in Athena. Dataset creation is disabled by default. Datasets are
+  preloaded in Databricks instance that tests run against. If you need to create new database on the instance,
+  run your test with this var bound to true."
+  true #_false)
```

Comment:
This will be disabled when the changes are approved.

Reply:
It would be good to know `why` this is done. 

Athena does it because `You cannot delete a [non-Iceberg] Athena database using DDL statements. It has to be deleted manually from S3 and the Glue Console`

It would also be good to get Databrick's version of the comment in athena data driver:
`INSTRUCTIONS FOR MANUALLY DROPPING AND RECREATING A DATABASE`

Reply:
We've discussed it on QPD weekly sometime ago, whether to generated new datasets per job. Conclusion was that it is not necesasry, as this would make things slower and unnecessarily more complex. That's why I've _just closed_ the corresponding task #43857.

I'll add the instructions into comments shortly.